config/job_conf.xml

```diff
@@ -306,6 +314,7 @@
     <!-- ICL -->
     <tool id="batman-nmr" destination="batman-container"/>
     <tool id="power-analysis" destination="papy-container"/>
+	<tool id="bruker2batman" destination="bruker2batman-container"/>
```

Comment:
Same as above, this needs to follow the new type of setup explained in the previously mentioned link.